crossterm_utils/src/error.rs

```diff
@@ -40,26 +43,17 @@ impl Display for ErrorKind {
     }
 }
 
-impl From<io::Error> for ErrorKind {
-    fn from(e: io::Error) -> ErrorKind {
-        ErrorKind::IoError(e)
-    }
-}
-
-impl From<fmt::Error> for ErrorKind {
-    fn from(e: fmt::Error) -> ErrorKind {
-        ErrorKind::FmtError(e)
-    }
-}
-
-impl From<std::string::FromUtf8Error> for ErrorKind {
-    fn from(e: std::string::FromUtf8Error) -> Self {
-        ErrorKind::Utf8Error(e)
-    }
+macro_rules! impl_from {
```

Comment:
Maybe we can move this to crossterm_utils/macros

I prefer to have all macros at the same place and it might be usefull in other parts of crossterm as well

Reply:
Moved.